manifests/plugin/disk.pp

```diff
@@ -15,7 +15,7 @@
   validate_bool($ignoreselected)
 
   if $::osfamily == 'RedHat' {
-    if $manage_package {
+    if $manage_package != undef {
```

Comment:
Write 'false' in hiera result into false, not true. I don't have great knowledge on spec test but I will try it.